sdk/eventhub/event-hubs/test/public/node/client.spec.ts

```diff
@@ -86,45 +90,42 @@ testWithServiceTypes((serviceVersion) => {
       // Extra check involving actual call to the service to ensure this works
       const hubInfo = await client.getEventHubProperties();
       should.equal(hubInfo.name, client.eventHubName);
-
-      await client.close();
     });
 
     describe("tracing", () => {
-      let tracer: TestTracer;
-      before(() => {
-        tracer = setTracer();
-      });
-
-      after(() => {
-        resetTracer();
-      });
-
       it("getEventHubProperties() creates a span with a peer.address attribute as the FQNS", async () => {
-        const client = new EventHubConsumerClient(
+        client = new EventHubConsumerClient(
           EventHubConsumerClient.defaultConsumerGroupName,
           endpoint,
           env.EVENTHUB_NAME,
           credential
         );
         should.equal(client.fullyQualifiedNamespace, endpoint);
 
-        // Extra check involving actual call to the service to ensure this works
-        const hubInfo = await client.getEventHubProperties();
-        should.equal(hubInfo.name, client.eventHubName);
+        const withSpanStub = Sinon.spy(tracingClient, "withSpan");
 
-        await client.close();
+        // Ensure tracing is implemented correctly
+        await assert.supportsTracing(
+          (options) => client.getEventHubProperties(options),
+          ["ManagementClient.getEventHubProperties"]
+        );
 
-        const spans = tracer
-          .getKnownSpans()
-          .filter((s) => s.name === "Azure.EventHubs.getEventHubProperties");
+        // Additional validation that we created the correct initial span options
```

Comment:
Note: I would rather not mix-and-match stubs with our `supportsTracing` test util, instead we should be able to pass this information as options. But I don't have bandwidth to do that now so I will likely file and issue to go back to that.